apps/web/src/components/common/form-input.tsx

```diff
@@ -39,7 +45,12 @@ export function FormInput<T extends FieldValues>({
             <Input icon={icon} className={className} {...field} {...props} />
           </FormControl>
           <div className='min-h-[20px]'>
-            <FormMessage />
+            <FormMessage className={cn(
+              isFieldValid && "text-accent", // 성공시 accent 컬러
+              !isFieldValid && "text-destructive" // 실패시 destructive 컬러
+            )}>
+              {isFieldValid ? validMessage : undefined}
+            </FormMessage>
```

Comment:
```suggestion
           {isFieldValid &&(
              <FormMessage .../>
           )}
```

`isFieldValid` 할때 `FormMessage` 를 렌더링 하는 방식으로 가는건 어떨까요?